Lib/graphlib.py

```diff
@@ -132,14 +135,14 @@ def get_ready(self):
         return result
 
     def is_active(self):
-        """Return ``True`` if more progress can be made and ``False`` otherwise.
+        """Return True if more progress can be made and False otherwise.
 
-        Progress can be made if cycles do not block the resolution and either there
-        are still nodes ready that haven't yet been returned by "get_ready" or the
-        number of nodes marked "done" is less than the number that have been returned
-        by "get_ready".
+        Progress can be made if cycles do not block the resolution and either
+        there are still nodes ready to be returned by "get_ready()" or there
+        are nodes which were returned by "get_ready()" and which have not yet
+        been marked as "done()".
 
-        Raises ValueError if called without calling "prepare" previously.
+        Raises ValueError if called before "prepare()".
```

Comment:
This is personal, but I don’t like adding parentheses to all function names.  It is clear enough that these are function (or method) names, and the parens suggest that they can be called without params, which is often not true.  I don’t think there is a strong consensus about this for docstrings, so in this case I would preserve the existing style.  (CPython does not value style uniformity over other things.)